app/src/main/java/com/example/lhi06/residue/ItunesService.java

```diff
@@ -0,0 +1,38 @@
+package com.example.lhi06.residue;
+
+import android.support.annotation.NonNull;
+
+import java.io.IOException;
+
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public class ItunesService {
+
+    private static final String BASE_API_URL = "https://itunes.apple.com/";
```

Comment:
Possibly don't need the trailing forward slash as your @GET string starts with a forward slash